<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dc_viewerClass</class>
 <widget class="QMainWindow" name="dc_viewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dc_viewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="3">
     <widget class="QTreeView" name="fileTreeView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="XlLabelDisplay" name="colorLabel">
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="XlLabelDisplay" name="tempLabel">
      <property name="text">
       <string>N/A</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" rowspan="3">
     <widget class="XlPointCloudWidget" name="glWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="XlLabelDisplay" name="curvatureLabel">
      <property name="text">
       <string>Curvature</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="XlLabelDisplay" name="depthLabel">
      <property name="text">
       <string>Depth</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="XlLabelDisplay" name="normalLabel">
      <property name="text">
       <string>Normal</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="XlLabelDisplay" name="edgeLabel">
      <property name="text">
       <string>Edge</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSetRoot"/>
    <addaction name="actionSetTemppath"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSetRoot">
   <property name="text">
    <string>Set &amp;Root</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionSetTemppath">
   <property name="text">
    <string>Set &amp;Temppath</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>XlLabelDisplay</class>
   <extends>QLabel</extends>
   <header>xllabeldisplay.h</header>
  </customwidget>
  <customwidget>
   <class>XlPointCloudWidget</class>
   <extends>QWidget</extends>
   <header location="global">xlpointcloudwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="dc_viewer.qrc"/>
 </resources>
 <connections/>
</ui>
